### PR TITLE
Create  task_description.html

### PR DESCRIPTION
**Pull request by qkue**

### CODE REVIEW

**Comment by freemanlex** [comment]
Hello. While doing translation to any language, both files of "info" folder should be translated - "task_description" and "task_short_description". Short description is shown on the station page under the title of a mission. Please, update you pull request with additional files. Thanks.

**Comment by qkue** [comment]
> Hello. While doing translation to any language, both files of "info" folder should be translated - "task_description" and "task_short_description". Short description is shown on the station page under the title of a mission. Please, update you pull request with additional files. Thanks.

Hello, Got it, thanks, I'll change it.

**Comment by qkue** [comment]
@freemanlex Sorry, if you have time, take a look, if it's OK I'll fix the rest. Thanks!

**Comment by freemanlex** [comment]
Everything is OK!)

File: translations/ru/info/task_description.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!-- Describe task in general -->
+
+<p>Основная идея ООП - описать на языке программирования любой объект (в том числе и из реальной жизни), определив его характеристики, поведение и т.д.</p>
+<p>
+    Эта серия заданий поможет вам изучить основы ООП в Python, такие как создание классов, атрибутов и методов, наследование, переопределение методов, использование super() и полиморфизм. Если вам нужна дополнительная помощь, вы можете воспользоваться этой <a target="_blank" href="https://devdocs.io/python~3.11/tutorial/classes#a-first-look-at-classes">this</a> или другой документацией.
```
Comment (freemanlex) [inline translations/ru/info/task_description.html:5]: I will change it by myself - no need to "this" if you have "этой" )

Comment (qkue) [reply translations/ru/info/task_description.html:5]: Thank you! :)

**Pull request merged**